app/views/config-editor.html

```diff
@@ -0,0 +1,11 @@
+<div class="component-label section-label">Config Info
+  <button class="clipboard-button pull-right btn btn-default" data-clipboard-text="{{prettyConfig}}" ng-click="copy()">Copy Config</button>
+</div>
+<div ui-ace="{
```

Comment:
Can these be set on the controller?

Reply:
I did it this way, as this is how it is done elsewhere in the code-base. e.g.
https://github.com/aerogear/origin-web-console/blob/master/app/views/browse/build-config.html#L339